lib/apipie/validator.rb

```diff
@@ -25,8 +25,11 @@ def inspect
       end
 
       def self.inherited(subclass)
+        BaseValidator.validators.unshift(subclass)
```

Comment:
Previously when subclassing a concrete validator, e.g. `EnumValidator`, this method would init the `@validators` ivar on `EnumValidator` and insert the new subclass there.

By explicitly referencing `BaseValidator`, the new class is added to the correct list.

This change shouldn't affect backwards compatibility. Worst case if anyone subclassed existing validators, they probably did `BaseValidator.inherited(my_validator)` or so, in which case this fix would cause `my_validator` to appear in the list twice.